docs/sources/features/datasources/loki.md

```diff
@@ -92,6 +92,20 @@ Example queries:
 * `{name="kafka"} tsdb-ops.*io:2003`
 * `{instance=~"kafka-[23]",name="kafka"} kafka.server:type=ReplicaManager`
 
+## Live Tailing
+
+To view your logs live as they are added, choose `Live` from the refresh dropdown, and you should see your logs be displayed in real time.
+
+Note: This feature is only available in Grafana v6.3+
+
+## Log Context
+
+When using a search expression as detailed above, you now have the ability to retrieve the context surrounding your filtered results.
+By clicking the `Show Context` link on the filtered rows, you'll be able to investigate the log messages that came before and after the
+log message you're interested in.
+
+Note: This feature is only available in Grafana v6.3+
```

Comment:
In other .md files the compat note is under the header of the paragraph, formatted as a quote:
```
> Note: .....
```